packages/client/src/components/EntranceModal/RegistrationForm/RegistrationForm.tsx

```diff
@@ -0,0 +1,105 @@
+import { Button, TextField } from '@mui/material';
+import React from 'react';
+
+import { DEFAULT_FORM_DATA } from './constants';
+import { useStyles } from './useStyles';
+
+import { AuthApiSignUp } from '../../../services/api/useAuthApi';
+import useAuthController from '../../../services/controllers/useAuthController';
+
+const RegistrationForm = () => {
+  const classes = useStyles();
+  const { signUpController } = useAuthController();
+  const [formData, setFormData] = React.useState<AuthApiSignUp>(DEFAULT_FORM_DATA);
+
+  const signUpHandler = (e: React.FormEvent) => {
+    e.preventDefault();
+    signUpController(formData);
+  };
+
+  const changeInputDataHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const name = e.target.name;
+    const value = e.target.value;
+    setFormData(prevState => ({ ...prevState, [name]: value }));
+  };
+
+  return (
+    <form onSubmit={signUpHandler}>
+      <TextField
+        name="first_name"
+        value={formData.first_name}
+        onChange={changeInputDataHandler}
+        color={'secondary'}
+        className={classes.input}
+        label={'Name'}
+        required
+        fullWidth
+      />
+      <TextField
+        name="second_name"
+        value={formData.second_name}
+        onChange={changeInputDataHandler}
+        color={'secondary'}
+        className={classes.input}
+        label={'Lastname'}
+        required
+        fullWidth
+      />
+      <TextField
+        name="login"
+        value={formData.login}
+        onChange={changeInputDataHandler}
+        color={'secondary'}
+        className={classes.input}
+        label={'Login'}
+        required
+        fullWidth
+      />
+      <TextField
+        name="email"
+        value={formData.email}
+        onChange={changeInputDataHandler}
+        color={'secondary'}
+        className={classes.input}
+        label={'Email'}
+        required
+        fullWidth
+      />
+      <TextField
+        name="phone"
+        value={formData.phone}
+        onChange={changeInputDataHandler}
+        color={'secondary'}
+        className={classes.input}
+        label={'Phone'}
+        required
+        fullWidth
+      />
+      <TextField
+        name="password"
```

Comment:
```suggestion
        name="password"
        type="password"
```

ну и для поля повторить пароль + на странице авторизации

Reply:
Еще не делал никакой валидации. Хотел оставить это на отдельный таск